browser-test/bin/wait_for_server_start_and_run_tests.sh

```diff
@@ -18,4 +18,19 @@ echo detected server start
 
 ./bin/truncate_tables.sh
 
-yarn test $@
+debug=0
+for arg in "$@"; do
+    case $arg in
+        -d|--debug)
+        debug=1
+        shift # Remove --debug from processing
```

Comment:
does this mean --debug must come before other arguments?

Reply:
Specifically the `shift` line? Hmm let me test this to double check. I wouldn't think so though.

Reply:
You're right about `shift`. I fixed and manually tested it.